ofc-tests-runner/expected_test_results/nist_expected_results

```diff
@@ -0,0 +1,193 @@
+FM001.FOR:pass
+FM002.FOR:pass
+FM003.FOR:pass
+FM004.FOR:pass
+FM005.FOR:pass
+FM006.FOR:pass
+FM007.FOR:pass
+FM008.FOR:pass
+FM009.FOR:pass
+FM010.FOR:pass
+FM011.FOR:pass
+FM012.FOR:pass
+FM013.FOR:pass
+FM014.FOR:pass
+FM016.FOR:pass
+FM017.FOR:pass
+FM018.FOR:pass
+FM019.FOR:pass
+FM020.FOR:pass
+FM021.FOR:pass
+FM022.FOR:pass
+FM023.FOR:pass
+FM024.FOR:pass
+FM025.FOR:pass
+FM026.FOR:pass
+FM028.FOR:pass
+FM030.FOR:pass
+FM031.FOR:pass
+FM032.FOR:pass
+FM033.FOR:pass
+FM034.FOR:pass
+FM035.FOR:pass
+FM036.FOR:pass
+FM037.FOR:pass
+FM038.FOR:pass
+FM039.FOR:pass
+FM040.FOR:pass
+FM041.FOR:pass
+FM042.FOR:pass
+FM043.FOR:pass
+FM044.FOR:pass
+FM045.FOR:pass
+FM050.FOR:pass
+FM056.FOR:pass
+FM060.FOR:pass
+FM061.FOR:pass
+FM062.FOR:pass
+FM080.FOR:pass
+FM097.FOR:pass
+FM098.FOR:pass
+FM099.FOR:pass
+FM100.FOR:pass
+FM101.FOR:pass
+FM102.FOR:pass
+FM103.FOR:pass
+FM104.FOR:pass
+FM105.FOR:pass
+FM106.FOR:pass
+FM107.FOR:pass
+FM108.FOR:pass
+FM109.FOR:pass
+FM110.FOR:pass
+FM111.FOR:pass
+FM200.FOR:pass
+FM201.FOR:pass
+FM202.FOR:pass
+FM203.FOR:pass
+FM204.FOR:pass
+FM205.FOR:pass
+FM251.FOR:pass
+FM252.FOR:pass
+FM253.FOR:pass
+FM254.FOR:pass
+FM255.FOR:pass
+FM256.FOR:pass
+FM257.FOR:pass
+FM258.FOR:pass
+FM259.FOR:pass
+FM260.FOR:pass
+FM261.FOR:pass
+FM300.FOR:pass
+FM301.FOR:pass
+FM302.FOR:pass
+FM306.FOR:pass
+FM307.FOR:pass
+FM308.FOR:pass
+FM311.FOR:pass
+FM317.FOR:pass
+FM328.FOR:pass
+FM351.FOR:pass
+FM352.FOR:pass
+FM353.FOR:pass
+FM354.FOR:pass
+FM355.FOR:pass
+FM356.FOR:pass
+FM357.FOR:pass
+FM359.FOR:pass
+FM360.FOR:pass
+FM361.FOR:pass
+FM362.FOR:pass
+FM363.FOR:pass
+FM364.FOR:pass
+FM368.FOR:pass
+FM369.FOR:pass
+FM370.FOR:pass
+FM371.FOR:pass
+FM372.FOR:pass
+FM373.FOR:pass
+FM374.FOR:pass
+FM375.FOR:pass
+FM376.FOR:pass
+FM377.FOR:pass
+FM378.FOR:pass
+FM379.FOR:pass
+FM401.FOR:pass
+FM402.FOR:pass
+FM403.FOR:pass
+FM404.FOR:pass
+FM405.FOR:pass
+FM406.FOR:pass
+FM407.FOR:pass
+FM411.FOR:pass
+FM413.FOR:pass
+FM500.FOR:pass
+FM503.FOR:pass
+FM506.FOR:pass
+FM509.FOR:pass
+FM514.FOR:pass
+FM517.FOR:pass
+FM520.FOR:pass
+FM700.FOR:pass
+FM701.FOR:pass
+FM710.FOR:pass
+FM711.FOR:pass
+FM715.FOR:pass
+FM718.FOR:pass
+FM719.FOR:pass
+FM722.FOR:pass
+FM800.FOR:pass
+FM801.FOR:pass
+FM802.FOR:pass
+FM803.FOR:pass
+FM804.FOR:pass
+FM805.FOR:pass
+FM806.FOR:pass
+FM807.FOR:pass
+FM808.FOR:pass
+FM809.FOR:pass
+FM810.FOR:pass
+FM811.FOR:pass
+FM812.FOR:pass
+FM813.FOR:pass
+FM814.FOR:pass
+FM815.FOR:pass
+FM816.FOR:pass
+FM817.FOR:pass
+FM818.FOR:pass
+FM819.FOR:pass
+FM820.FOR:pass
+FM821.FOR:pass
+FM822.FOR:pass
+FM823.FOR:pass
+FM824.FOR:pass
+FM825.FOR:pass
+FM826.FOR:pass
+FM827.FOR:pass
+FM828.FOR:pass
+FM829.FOR:pass
+FM830.FOR:pass
+FM831.FOR:pass
+FM832.FOR:pass
+FM833.FOR:pass
+FM834.FOR:pass
+FM900.FOR:pass
+FM901.FOR:pass
+FM903.FOR:pass
+FM905.FOR:pass
+FM906.FOR:pass
+FM907.FOR:fail
```

Comment:
I though we passed all the nists test

Reply:
This test also fails when compiled&executed with OFC.
When it executes, the output says the formatting can vary depending on some circumstances. 
This test has .expected file provided in OFC tests with expected output, which is exactly the same as our's (and OFC's), but the formatting is slightly different.
To make it 'pass' here, the .expected file in OFC tests would need to be replaced with the one with our/OFC's formatting.

Reply:
Ok, maybe check why this fails, I though he said that all the nist tests passed in OFC